pkg/steps/artifacts.go

```diff
@@ -366,27 +366,6 @@ func addPodUtils(pod *coreapi.Pod, artifactDir string, decorationConfig *prowv1.
 	return nil
 }
 
-func addArtifacts(pod *coreapi.Pod, artifactDir string) {
```

Comment:
This _looks_ super suspect, but it turns out while Template code does the same thing, it uses a separate set of methods to do it. Sigh.